build/commands/lib/util.js

```diff
@@ -880,6 +903,21 @@ const util = {
     util.run('git', ['fetch', 'origin', ref.replace(/^origin\//, '')], options)
     util.run('git', ['-c', 'advice.detachedHead=false', 'checkout', 'FETCH_HEAD'], options)
   },
+
+  writeFileIfModified: (filePath, content) => {
+    if (
+      !fs.existsSync(filePath) ||
+      fs.readFileSync(filePath, { encoding: 'utf-8' }) !== content
+    ) {
+      const dirName = path.dirname(filePath)
```

Comment:
Maybe use `ensureDir/ensureFile` from `fs-extra`?
https://github.com/jprichardson/node-fs-extra/blob/master/docs/ensureFile.md

Reply:
added `ensureFile`, didn't know about it. thanks!